Firestore/Source/Public/FIRDocumentSnapshot.h

```diff
@@ -21,6 +21,49 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ * Controls the return value for server timestamps that have not yet been set to
+ * their final value.
+ */
+typedef NS_ENUM(NSInteger, FIRServerTimestampBehavior) {
```

Comment:
The default behavior should be available here as one of the enumeration values and the comments should reflect that it's the default.

Also, I wonder if we should name it `...Null` rather than `...Default`. The behavior itself is to return a null value for server timestamps; it just happens to be the default behavior.

Reply:
This is one of the discussion items for the API discussion tomorrow. I will adapt this PR as needed.

Reply:
nit: slight preference to put the intended default case of the enum first

Reply:
Done.